.circleci/config.yml

```diff
@@ -69,4 +69,6 @@ workflows:
     jobs:
       - build:
           <<: *always_run
-          context: sidekiq-enterprise
+          context:
```

Comment:
https://app.circleci.com/pipelines/github/doximity/simplekiq/92/workflows/ae3cfd0b-23a3-495e-bce3-40c98aa0c8dd/jobs/161/tests#failed-test-0

You got bundle install working now